theurgy.lic

```diff
@@ -55,75 +135,296 @@ def buy_wine
   put_in_container('wine')
 end
 
+def buy_oil
+  DRCI.buy_item(19_076, 'oil')
+  put_in_container('oil')
+end
+
+def buy_parchment
+  DRCT.walk_to('5735')
+  fput 'read placard'
+  pause 1
+  fput 'order 4 from monk'
+  fput 'unroll my parchment'
+  put_in_container('golden parchment')
+end
+
+def buy_Taffelberries
+  DRCI.order_item(8280, 14)
+  fput 'get Taffelberries'
```

Comment:
`order_item` leaves the items in hand. Why do you need to get them? Do they go to the ground after buying?